app/javascript/src/PaymentGateways/stripe/styles/stripe.scss

```diff
@@ -1,11 +1,14 @@
 // scss-lint:disable IdSelector
 $white: white;
+$black: black;
```

Comment:
I am not sure but this might be redundant as black is the default color.

Reply:
Well, parent containers may set a different `color` (other than black), so I think it's safer to keep it explicit.